editor/app/(workbench)/[org]/[proj]/[id]/connect/channels/page.tsx

```
@@ -51,56 +45,347 @@ import {
+                    {subject_template?.trim() ||
+                      "Your submission has been received"}
+                  </EmailFrameSubject>
+                  <EmailFrameSender
+                    name={from_name?.trim() || "Grida Forms"}
+                    email="no-reply@accounts.grida.co"
+                    date="Just now"
+                  />
+                  <EmailFrameBody className="prose prose-stone dark:prose-invert max-w-none">
+                    {body_html_template?.trim() ? (
+                      <div
+                        dangerouslySetInnerHTML={{
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)